cmd/performance-profile-creator/cmd/root.go

```diff
@@ -35,8 +35,11 @@ import (
 
 var (
 	validTMPolicyValues = []string{kubeletconfig.SingleNumaNodeTopologyManager, kubeletconfig.BestEffortTopologyManagerPolicy, kubeletconfig.RestrictedTopologyManagerPolicy}
-	// TODO: Explain the power-consumption-mode and associated kernel args
+	// default => no args
+	// low-latency => "nmi_watchdog=0", "audit=0",  "mce=off"
+	// ultra-low-latency: low-latency values + "processor.max_cstate=1", "intel_idle.max_cstate=0", "idle=poll"
```

Comment:
It would be nice to link https://gist.github.com/wmealing/2dd2b543c4d3cff6cab7 using a comment, but we can do in a later PR.